CONTRIBUTING.md

```diff
@@ -331,93 +330,6 @@ satisfy the tests provided for each problem.
 You can include advice and helpful links for your language track in the
 `SETUP.md` file.
 
-## Maintaining a Track
```

Comment:
> which information you identified

My point wasn't that you hadn't done a thorough review, but rather that you had clearly identified something missing (per your comment), but left it to me to go figure out what it was you had discovered. I would have appreciated more detail about what you had seen.

I fully accept that it was up to me to do a thorough walk through to make sure that everything was included.